### DISCUSSION TITLE
Default color values of aliasing colors are missing

### DISCUSSION THREAD

**Discussion by ghost** [tailwindlabs/tailwindcss#9177] [Help] [answered]
Hi,

in the documentation there is a section about aliasing colors:
https://tailwindcss.com/docs/customizing-colors#aliasing-color-names

After applying this at our project, the **default** color value of an aliased color was not accessible!

I found a cumbersome workaround by adding the ``DEFAULT`` color value manually to a copy of the referenced color, but imho that feels odd:
```js
theme: {
        extend: {
            colors: {
		// ``bg-dangerous`` wont work with this! But ``bg-rose`` does!?
                dangerous: colors.rose
		// With this workaround accessing the default color is again possible
                dangerous: { ...colors.rose, DEFAULT: colors.rose[500] },
            }
        },
    },
```

Any ideas about this?

**Comment by wongjn** [comment, marked as answer]
What version of Tailwind are you using? With default configuration, all colors have a color shade prefix – there is no `DEFAULT` key and thus, colors like `bg-rose` should **not** work – see this [Tailwind Play](https://play.tailwindcss.com/0xpR5UPAWv?file=config) for your example.

If you *do* want a default color shade, then you can do what you are doing now.

**Reply by ghost** [reply]
I am using tailwindcss 3.x (exact version is not accassible right now; I can provide this tomorrow).

We are using the tailwind components and there you will find classes like ``text-white``, for example here:
https://tailwindui.com/components/application-ui/elements/buttons
(In order to see the code, you must have an account of course)

So how is it possible? (And it works!)
```HTML
<div class="bg-black text-white">Hello World</div>
```
I just tried this in your example.

**Reply by wongjn** [reply]
`bg-black` and `bg-white` work because the `white` and `black` colors don't have shades and are literal color values:
https://github.com/tailwindlabs/tailwindcss/blob/7f93550e635f58dfcc9aa97544681e76aa177b77/src/public/colors.js#L14-L15

Whereas if you compare with other colors that do:
https://github.com/tailwindlabs/tailwindcss/blob/7f93550e635f58dfcc9aa97544681e76aa177b77/src/public/colors.js#L16-L27

**Reply by ghost** [reply]
Ah ok... thx.

Now I have understood the differences between those!
